main.js

```diff
@@ -2,15 +2,24 @@ import openMenu from './js/hamburguer.js';
 import createCardProject from './js/works.js';
 import createModal from './js/modalWindow.js';
 import validation from './js/validacionForm.js';
+import { getInfo, setInfo } from './js/localStorage.js';
 
 document.addEventListener('DOMContentLoaded', () => {
   createCardProject();
   createModal();
-  validation();
+  getInfo();
 });
 
 document.documentElement.addEventListener('click', (e) => {
   if ((e.target.matches('.menu-mobil a')) || (e.target.matches('.header-menu-hamburguer *'))) {
     openMenu();
   }
 });
+
+document.getElementById('myForm').addEventListener('submit', (e) => {
```

Comment:
- [ ] Kindly consider changing the event listener from `submit` to `change` in order to fulfill the following requirement. because the input data must be saved in the local storage on input change and not on form submission.
   -  When the user changes the content of any input field, the data is saved to the local storage.

Reply:
done